packages/docs/src/components/SDKSection.tsx

```diff
@@ -1,11 +1,19 @@
 import { cssObj } from '@fuel-ui/css';
 import { Box, Button, Image, Text } from '@fuel-ui/react';
-import { useRouter } from 'next/navigation';
+import { usePathname, useRouter } from 'next/navigation';
 
 import connectorsPreviewImg from '../../public/connectors-preview.png';
 
 export function SDKSection() {
   const { push } = useRouter();
+  const pathname = usePathname();
+
+  // Allows routing to work regardless of path structure (e.g. docs hub)
+  function routeToGettingStarted() {
+    const basePath = pathname.split('/').slice(0, -2).join('/');
+    const newCompletePath = `${basePath}/dev/getting-started/`;
```

Comment:
I think to keep it really stable we might get it from `window.location.host`
otherwise it will break if we change the URL structure for something different from "2" level of folders